transformer/kubernetes/clusterselector.go

```diff
@@ -29,6 +29,10 @@ import (
 )
 
 const (
+	// clusterTypeKey is the key for QA ID
+	clusterTypeKey = "clustertype"
+	// defaultClusterSelectorSuffix defines the default suffix for the QA IDs
+	defaultClusterSelectorSuffix = "cs"
```

Comment:
Do we need a default one? What if we make it empty by default?

Reply:
done